src/platforms/apple/guides/watchos/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: watchOS
+---
+
+## Installation {#install}
+
+<PlatformContent includePath="getting-started-install" />
+
+For other installation methods, please see our [documentation](https://docs.sentry.ip/platforms/apple/install/).
+
+## Configuration {#configure}
+
+<PlatformContent includePath="getting-started-config" />
+
+## Debug Symbols {#debug-symbols}
+
+Before capturing crashes, you need to provide debug information to Sentry. Debug information is provided by uploading dSYM files using one of two methods, dependent on your setup:
+
+- [With Bitcode](dsym/#dsym-with-bitcode)
+- [Without Bitcode](dsym/#dsym-without-bitcode)
+
+## Verify
+
+To validate that the SDK is correctly set up and connected to your Sentry project, capture a test message:
+
+<PlatformContent includePath="capture-message" />
+
+If everything is working correctly, this message should appear within seconds in your Sentry project.
+
+## Crash Handling
+
+Our SDK hooks into all signal and exception handlers, except for MacOS. If you are using MacOS, please see the additional step provided in [Advanced Usage](usage/)
+To try it out, the SDK provides a test crash function:
+
+<PlatformContent includePath="trigger-crash" />
+
+_If you crash with a debugger attached, nothing will happen._
+
+Crashes are submitted only upon re-launch of the application. To view the crash in Sentry, close your app and re-launch it.
```

Comment:
This page should describe what limitations we have with watchOS symbolication, and how those affect all crash reporting tools and not only Sentry.
I don't know it well enough to write this well. @mitsuhiko @HazAT ?

Reply:
https://app.asana.com/0/1162483910998154/1185636138279961

Some note was added here: https://github.com/getsentry/sentry-docs/pull/1924/files
Is it lost?

Would be nice to have a better technical explanation what this means